### PR TITLE
Update/Remove Packages

### PR DESCRIPTION
**Pull request by SpicyGarlicAlbacoreRoll**
- Updates several library packages
- Removes several deprecated/unused packages
- modifies tsconfig

### CODE REVIEW

File: src/app/services/saved-search.service.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { Injectable } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { combineLatest, Observable } from 'rxjs';
 import { map, switchMap } from 'rxjs/operators';
-import * as uuid from 'uuid/v1';
+// import * as uuid from 'uuid/v1';
```
Comment (tylercchase) [inline src/app/services/saved-search.service.ts]:
```suggestion

```

**Review by tylercchase — CHANGES_REQUESTED** [review]
I get this error when loading the site. Not sure where exactly it's coming from 
image

File: src/app/components/results-menu/results-menu.component.html [hunk 2]
```diff
@@ -6,18 +6,18 @@
...
       id="result-div"
       class="w100"
-      [resizeEdges]="{ top: !(imageViewerOpen$ | async) }"
+
       [ngClass]="{'mobile-size': breakpoint === breakpoints.MOBILE}"
       [style.height.px]="menuHeightPx"
       mwlResizable
       [validateResize]="validate"
       [enableGhostResize]="true"
-      [resizeCursorPrecision]="20"
```
Comment (tylercchase) [inline src/app/components/results-menu/results-menu.component.html:15]: This gets rid of the cursor on the edge of the results menu.

Comment (SpicyGarlicAlbacoreRoll) [reply src/app/components/results-menu/results-menu.component.html:15]: resizeCursorPrecision is a deprecated field with the latest version of the package, fixed the missing cursor with 
[27b7f12](https://github.com/asfadmin/Discovery-SearchUI/pull/1551/commits/27b7f12e2253d701116ea054cd4e800c4c17197e)

**Pull request merged**